static/src/js/components/ContactInfo/ContactInfo.scss

```diff
@@ -1,4 +1,4 @@
-fieldset.account-form {
+.contact-info-form {
```

Comment:
This looks much better. Only change I would make would be getting rid of these nested element selectors. They are creating unnecessary specificity.

```
.contact-info-form {
  ...
  > ul {
    ...
    > li {
       ...
    }
  }
}
```
should become something like
```
.contact-info-form {
  ...
  &__list {
    ...
  }
  &__item {
      ...
  }
}
```